dts/bindings/bluetooth/nordic,bt-cs-antenna-switch.yaml

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+  This is an abstract representation of a generic antenna switching circuit
+  which is controlled by GPIO pins. Up to four antenna ports (ANT1-4) are defined.
+
+compatible: "nordic,bt-cs-antenna-switch"
+
+include: base.yaml
+
+properties:
+  multiplexing-mode:
+    type: int
+    description: |
+      Multiplexing mode. This setting determines whether or not the switching circuit
+      is controlled using a one-to-one mapping between GPIOs and antennas.
```

```suggestion
      is controlled using a one-to-one mapping between pins and antennas.
```
i would suggest to replace `GPIO` to `pins` or `GPIO pins` in the whole file  for better clarity